### PR TITLE
[u-mr1] Fix properties that are missing a vendor prefix

### PR DESCRIPTION
**Pull request by bartcubbins**
```
[ 75% 305/402 2m6s remaining] //system/sepolicy/contexts:vendor_property_contexts checking namespace of vendor_property_contexts [common]
FAILED: out/soong/.intermediates/system/sepolicy/contexts/vendor_property_contexts/android_common/pdx201/vendor_property_contexts_namespace_checked
out/host/linux-x86/bin/check_prop_prefix --property-contexts out/soong/.intermediates/system/sepolicy/contexts/vendor_property_contexts/android_common/pdx201/vendor_property_contexts --allowed-property-prefix ctl.odm. --allowed-property-prefix ctl.vendor. --allowed-property-prefix 'ctl.start$odm.' --allowed-property-prefix 'ctl.start$vendor.' --allowed-property-prefix 'ctl.stop$odm.' --allowed-property-prefix 'ctl.stop$vendor.' --allowed-property-prefix init.svc.odm. --allowed-property-prefix init.svc.vendor. --allowed-property-prefix ro.boot. --allowed-property-prefix ro.hardware. --allowed-property-prefix ro.odm. --allowed-property-prefix ro.vendor. --allowed-property-prefix odm. --allowed-property-prefix persist.odm. --allowed-property-prefix persist.vendor. --allowed-property-prefix vendor. --allowed-property-prefix persist.camera. --strict && cp -f out/soong/.intermediates/system/sepolicy/contexts/vendor_property_contexts/android_common/pdx201/vendor_property_contexts out/soong/.intermediates/system/sepolicy/contexts/vendor_property_contexts/android_common/pdx201/vendor_property_contexts_namespace_checked # hash of input list: 65ad390488da097f583d7a675d0dabeff205b85ac6fb35d11de934cece72eeef
******************************
10 violations found:
net.r_rmnet_data0               u:object_r:net_rmnet_prop:s0
rild.                           u:object_r:radio_prop:s0
ro.wifi.                        u:object_r:vendor_wifi_prop:s0
sensors.                        u:object_r:sensors_prop:s0
sys.keymaster.loaded            u:object_r:keymaster_prop:s0
sys.listeners.registered        u:object_r:tee_listener_prop:s0
wc_transport.                   u:object_r:wc_prop:s0
persist.somc.cust.              u:object_r:vendor_somc_cust_prop:s0
persist.somc.modemswitcher.     u:object_r:vendor_somc_modemswitcher_prop:s0
sony.modemswitcher.             u:object_r:vendor_somc_modemswitcher_prop:s0
******************************
vendor's and odm's property_contexts MUST use ONLY vendor-prefixed properties.
This is enforced by VTS, so please fix such offending properties.
Allowed property prefixes for out/soong/.intermediates/system/sepolicy/contexts/vendor_property_contexts/android_common/pdx201/vendor_property_contexts: ['ctl.odm.', 'ctl.vendor.', 'ctl.start$odm.', 'ctl.start$vendor.', 'ctl.stop$odm.', 'ctl.stop$vendor.', 'init.svc.odm.', 'init.svc.vendor.', 'ro.boot.', 'ro.hardware.', 'ro.odm.', 'ro.vendor.', 'odm.', 'persist.odm.', 'persist.vendor.', 'vendor.', 'persist.camera.']
You can temporarily disable this check with setting BUILD_BROKEN_VENDOR_PROPERTY_NAMESPACE := true in BoardConfig.mk.
But property namespace is enforced by VTS, and you will need to fix such violations to pass VTS.
See test/vts-testcase/security/system_property/vts_treble_sys_prop_test.py for the detail of the VTS.
05:04:01 ninja failed with: exit status 1
```

### CODE REVIEW

File: vendor/vendor_init.te [hunk 1]
```diff
@@ -1,12 +1,9 @@
 typeattribute vendor_init data_between_core_and_vendor_violators;
 
-# TODO(b/compatible): Remove once all devices use compatible props
 # Get vendor.rild.rilpath
 get_prop(vendor_init, vendor_radio_prop)
 # Set rild.rilpath
-not_compatible_property(`
-  set_prop(vendor_init, radio_prop)
-')
+set_prop(vendor_init, vendor_radio_prop)
```
Comment (MarijnS95) [inline vendor/vendor_init.te:6]: Shouldn't the comment be `Set vendor.rild.rilpath` now?

File: vendor/hal_bluetooth_default.te [hunk 2]
```diff
@@ -9,7 +9,6 @@ allow hal_bluetooth_default kernel:system module_request;
 allow hal_bluetooth_default self:qipcrtr_socket create_socket_perms_no_ioctl;
 allow hal_bluetooth_default self:socket create;
 
-set_prop(hal_bluetooth_default, wc_prop)
 set_prop(hal_bluetooth_default, bluetooth_prop)
```
Comment (MarijnS95) [inline vendor/hal_bluetooth_default.te:12]: No complaints about a `set` for a seemingly non-vendor prop here? At least the server is **not**-`neverallow`ed to set it: https://cs.android.com/android/platform/superproject/+/android14-qpr3-release:system/sepolicy/private/property.te;l=257;drc=9f41fc081fd282948ba2257bf025803f5868b8c0

**Review by MarijnS95 — APPROVED** [review]
Yeah it looks nice to remove all non-`vendor.` violators. Didn't check if everything has a good fallback and/or is migrated, but changes look solid.

File: vendor/hal_bluetooth_default.te (hunk 2 above)
Comment (bartcubbins) [reply vendor/hal_bluetooth_default.te:12]: good catch. nothing in the log, but I will remove it during rebase

**Review by jerpelea — APPROVED** [review]

**Pull request merged**